### PR TITLE
print roles as default user subtitle

### PR DESCRIPTION
**Pull request by G3z**
I know this is a minor thing and users can easily customize this but still I think this is a cool default

## Proposed Changes
roles as default subtitle for userpresenter

### CODE REVIEW

**Comment by tabuna** [comment]
I think between `Regular user` and `Plain user` at the request of Google, the first phrase is more common

File: install-stubs/Orchid/Presenters/UserPresenter.php [hunk 1]
```diff
@@ -32,7 +32,11 @@ public function title(): string
      */
     public function subTitle(): string
     {
-        return 'Administrator';
```
Comment (tabuna) [inline install-stubs/Orchid/Presenters/UserPresenter.php:35]: What do you think of this change?

File: install-stubs/Orchid/Presenters/UserPresenter.php [hunk 2]
```diff
@@ -32,7 +32,11 @@ public function title(): string
...
+        $roles = $this->entity->roles->pluck('name')->implode(' / ');
```
Comment (tabuna) [inline install-stubs/Orchid/Presenters/UserPresenter.php:35]: What do you think of this change?

Comment (G3z) [reply install-stubs/Orchid/Presenters/UserPresenter.php:35]: Fine by me!

**Comment by G3z** [comment]
i'll add the `?? 'Regular user'` fallback

**Comment by tabuna** [comment]
Adding `?? 'Regular user'` does not work as an empty string will be returned

**Pull request merged**